presto-native-execution/scripts/dockerfiles/prestissimo-runtime.dockerfile

```
@@ -46,6 +46,12 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# CLP S3 support currently needs ca-certificates package to access the presigned URL
+RUN apt-get update && \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)